src/Spout/Writer/Common/Entity/Cell.php

```diff
@@ -55,12 +57,29 @@ class Cell
     protected $type = null;
 
     /**
+     * The cell style
+     * @var Style|null
+     */
+    protected $style = null;
+
+    /**
+     * @var StyleMerger
+     */
+    protected $styleMerger;
+
+    /**
      * Cell constructor.
      * @param $value mixed
+     * @param $style Style
      */
-    public function __construct($value)
+    public function __construct($value, Style $style = null)
     {
         $this->setValue($value);
+        if ($style) {
```

Comment:
You can probably remove the `if` statement, as it's fine to set the style to `null`